src/components/VuetableRowHeader.vue

```diff
@@ -12,24 +12,24 @@
             :class="headerClass('vuetable-th-component', field)"
             :style="{width: field.width}"
             @vuetable:header-event="vuetable.onHeaderEvent"
-            @click="onColumnHeaderClicked(field, $event)"
+            @mousedown="onColumnHeaderClicked(field, $event)"
```

Comment:
sorting should only be fired on click and not on mousedown. Please keep the click event for that.

Reply:
Done.